gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -207,6 +213,16 @@
       this.partitionFilter = Optional.fromNullable(this.dataset.properties.getProperty(COPY_PARTITIONS_FILTER_CONSTANT));
     }
 
+    try {
+      this.fastPartitionSkip = this.dataset.properties.containsKey(FAST_PARTITION_SKIP_PREDICATE) ? Optional.of(
+          GobblinConstructorUtils.invokeFirstConstructor(
+              (Class<Predicate<Partition>>) Class.forName(this.dataset.properties.getProperty(FAST_PARTITION_SKIP_PREDICATE)),
+              Lists.<Object>newArrayList(this), Lists.newArrayList()))
```

Comment:
This will try two constructors, `<init>(HiveCopyEntityHelper)` and `<init>()`. In case someone uses a predicate that doesn't take arguments.